### PR TITLE
update queue api tests

### PR DESCRIPTION
**Pull request by muodov**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -677,7 +674,7 @@ A new item can be put in the Queue, this item will auto expire after 30 seconds.
 
         {
             "url": "http://www.example.com/",
-            "id": "fuSHr0sRQ1usugvheahwQ"
+            "id": "fasdasdasdasdasdasdas"
```
Comment (jsnjack) [inline apiary.apib]: Hmm... This is public documentation, it would be nice to have something more realistic here

File: apiary.apib [hunk 2]
```diff
@@ -652,14 +652,11 @@ A list of objects containing the following parameters:
 
 ### Add an item to the Queue [POST]
 
-A new item can be put in the Queue, this item will auto expire after 30 seconds. In order to make sure that your item stays alive you will need to repeat the request with the same ID.
```
Comment (jsnjack) [inline apiary.apib:655]: That statement about 30 seconds, it is still true. I think it is better to have it in the documentation

File: apiary.apib (hunk 1 above)
Comment (muodov) [reply apiary.apib]: yeah, I'll generate another session id

File: apiary.apib (hunk 2 above)
Comment (muodov) [reply apiary.apib:655]: It is true for sessions, but not for the queue. As long as the session exists and queued, it will be visible in queue

**Pull request merged**